### ISSUE TITLE
Need a clearer landmark and label for notebook output area

### ISSUE THREAD

**Issue by minsa110** [microsoft/vscode#168814]
Currently, the notebook cell output area is marked up as role = "list". This means all the notebook cell output is included in a list container. List is not a good way to indicate this important result area. I recommend using other effective landmark region semantic with a proper aria label so that screen reader users can more easily identify this area. also this landmark area needs to have a document role to auto-activate browse mode for screen readers.

Mentioned from: https://github.com/microsoft/vscode/issues/90408

**Comment by roblourens** [comment]
The only `role=list` is the actual list of notebook cells. Can you tell me which role the output container should have @minsa110?

**Comment by VSCodeTriageBot** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!

**Comment by Tyriar** [comment]
@jooyoungseo I'm not sure whether contenteditable will actually be better for screen reader users, we're actually planning on removing contenteditable by default when accessibility mode is _on_ and allow the user to just navigate it like a regular document. It will be contenteditable when accessibility mode is off to allow users to select via the keyboard, and then have a setting to allow the user to switch. We're eager to hear feedback on which is best for native screen reader users.

**Comment by jooyoungseo** [comment]
Aw, I see. @Tyriar -- I can test the new terminal output accessibility once it's released in insiders. I can also ask other blind programmers to test and share their thoughts to find the best configuration. However, I think this issue needs to be reopened and the lesson from the integrated terminal can contribute to this issue as well.

**Comment by DonJayamanne** [comment]
> and the lesson from the integrated terminal can contribute to this issue as well.

@Tyriar Any comments on the learnings or what can be applied here from the integrated terminal?

**Comment by Tyriar** [comment]
@meganrogge would be better to answer that

**Comment by meganrogge** [comment]
There is now an accessible view for notebook output (from @amunger), so this can be closed IMO. @jooyoungseo , pls lmk if you disagree. 

 <img width="862" alt="Image" src="https://github.com/user-attachments/assets/d437428f-713d-49af-bbda-7caafd2d0ebc" />
